load.php

```diff
@@ -503,6 +523,26 @@ function perflab_run_module_activation_deactivation( $old_value, $value ) {
 	return $value;
 }
 
+/**
+ * Handles dismissing a WordPress pointer.
+ *
+ * @since n.e.x.t
+ *
+ * @param WP_User $user The WP_User object.
+ */
+function perflab_dismissed_wp_pointers( $user ) {
+	$dismissed = array_filter( explode( ',', (string) get_user_meta( $user->ID, 'dismissed_wp_pointers', true ) ) );
+
+	if ( ! in_array( 'perflab-module-migration-pointer', $dismissed, true ) ) {
+		return;
+	}
+
+	unset( $dismissed[ array_search( 'perflab-module-migration-pointer', $dismissed, true ) ] );
```

Comment:
Slightly cleaner would be to only call `array_search()` which can be used alone to get the same information.
```suggestion
	$pointer_index = array_search( 'perflab-module-migration-pointer', $dismissed, true );
	if ( false === $pointer_index ) {
		return;
	}

	unset( $dismissed[ $pointer_index ] );
```